encrypter.cpp

```diff
@@ -0,0 +1,168 @@
+#include <iostream>
+#include <fstream>
+#include <regex>
+#include <string>
+#include <stdlib.h>
+#include <unistd.h>
+
+using namespace std;
+
+string read_file(string& address, string error_message, bool clean = false)
+{
+  ifstream f;
+
+  f.open(address);
+  if(!f)
+  {
+    cout << error_message;
+    f.close();
+    exit(1);
+  }
+
+  char c;
+  string text = "";
+
+
+  while (f.get(c))
+    if ((c != ' ') && (c != '\n') && (c != '.') || clean)
+      text += c;
+
+  f.close();
+
+  if (text == "")
+  {
+    cout << "Given file is empty";
+    exit(1);
+  }
+
+  return text;
+}
+
+void record_file(string& address, string& text)
+{
+  ofstream f;
+  f.open(address);
```

Comment:
You can open file in one line. Just: ofstream(address)
And address is a strange name. It's not an address it is a filename.